build/ruby22rhel7-template-docker.json

```diff
@@ -123,7 +123,7 @@
           "dockerStrategy": {
             "from": {
               "kind": "DockerImage",
-              "name": "centos/ruby-22-centos7:latest"
+              "name": "centos/ruby-25-centos7:latest"
```

Comment:
Please change to use openshift imagestream like above, if you use this image directly, the case will be failed in disconnected cluster; but if use openshift imagestream, we can mirror/import ruby image.